--- pkg/config/config.go.orig	2018-08-09 11:41:07.102186052 -0400
+++ pkg/config/config.go	2018-08-09 11:42:26.570626933 -0400
@@ -87,10 +87,10 @@
 // InitialDefaultSettings provides a DefaultSettings struct with sane values for interaction with the Automation Broker
 func InitialDefaultSettings() *DefaultSettings {
 	return &DefaultSettings{
-		BrokerNamespace:          "openshift-automation-service-broker",
+		BrokerNamespace:          "openshift-ansible-service-broker",
 		BrokerResourceURL:        "/apis/servicecatalog.k8s.io/v1beta1/clusterservicebrokers/",
-		BrokerRouteName:          "openshift-automation-service-broker",
-		ClusterServiceBrokerName: "openshift-automation-service-broker",
+		BrokerRouteName:          "asb",
+		ClusterServiceBrokerName: "ansible-service-broker",
 	}
 }
 
